src/legacy/core_plugins/kibana/server/field_formats/register.js

```diff
@@ -17,36 +17,36 @@
  * under the License.
  */
 
-import { createUrlFormat } from '../../common/field_formats/types/url';
-import { createBytesFormat } from '../../common/field_formats/types/bytes';
-import { createDateOnServerFormat } from '../../common/field_formats/types/date_server';
-import { createDateNanosFormat } from '../../common/field_formats/types/date_nanos';
-import { createRelativeDateFormat } from '../../common/field_formats/types/relative_date';
-import { createDurationFormat } from '../../common/field_formats/types/duration';
-import { createIpFormat } from '../../common/field_formats/types/ip';
-import { createNumberFormat } from '../../common/field_formats/types/number';
-import { createPercentFormat } from '../../common/field_formats/types/percent';
-import { createStringFormat } from '../../common/field_formats/types/string';
-import { createSourceFormat } from '../../common/field_formats/types/source';
-import { createColorFormat } from '../../common/field_formats/types/color';
-import { createTruncateFormat } from '../../common/field_formats/types/truncate';
-import { createBoolFormat } from '../../common/field_formats/types/boolean';
-import { createStaticLookupFormat } from '../../common/field_formats/types/static_lookup';
+import { UrlFormat } from '../../../../../plugins/data/common/field_formats/converters/url';
```

Comment:
What about to create index.ts file in../../../../../plugins/data/common/field_formats/converters/ folder and import all from it instead of adding a lot of  ../ in each line?

Reply:
The `index.ts` has been created.